graphql/schemabuilder/helpers.go

```diff
@@ -390,3 +405,111 @@ func castSlice(slice interface{}) []interface{} {
 
 	return ret
 }
+
+func (sb *schemaBuilder) buildPaginatedArgParser(originalArgType reflect.Type) (*argParser, graphql.Type, error) {
+
+	interfaceParser, interfaceType, err := sb.makeStructParser(originalArgType)
+	if err != nil {
+		return nil, nil, fmt.Errorf("failed to build args for paginated field")
+	}
+
+	typ := reflect.TypeOf(ConnectionArgs{})
+	fields := make(map[string]argField)
```

Comment:
could you describe what `fields` should be expected to contain?

Reply:
fields in the ConnectionArgs (except the interface field)